sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -18252,6 +18252,50 @@ public void testLeftJoinSubqueryWithSelectorFilter(Map<String, Object> queryCont
     );
   }
 
+  @Test
+  @Parameters(source = QueryContextForJoinProvider.class)
```

Comment:
Please share some context either here or in the PR description about why you are adding this test.